src/colab/server-picker.ts

```diff
@@ -94,16 +114,33 @@ export class ServerPicker {
     // Skip prompting for an accelerator for the default variant (CPU).
     if (state.variant === Variant.DEFAULT) {
       state.accelerator = 'NONE';
-      return (input: MultiStepInput) => this.promptForAlias(input, state);
+      const shapePicks = getShapePicksFromAccelerator(
+        shapesByAccelerators,
+        state.accelerator,
+      );
+      if (shapePicks.length <= 1) {
+        state.shape =
+          shapePicks.length === 1 ? shapePicks[0].value : Shape.STANDARD;
+        return (input: MultiStepInput) =>
+          this.promptForAlias(input, state, /** totalSteps= */ 2);
+      }
```

Comment:
Is it possible to consolidate / refactor the duplicate logic between here and in `promptForAccelerator`? Only difference is the `totalSteps` value.